pkg/omf/cli/omf.repo.pull.fish

```diff
@@ -8,11 +8,12 @@ function omf.repo.pull
   set -l repo_dir $argv[1]
 
   function omf.repo.git -V repo_dir
-    command git --git-dir "$repo_dir"/.git --work-tree "$repo_dir" $argv
+    command git -C "$repo_dir" $argv
```

Comment:
Apart from this change that would not support some versions of git (let's discuss this in a separate issue), I like the PR!